dcc-reissue-api.yaml

```diff
@@ -183,6 +183,26 @@ paths:
                   value:
                     error: RI400-1500
                     message: certificates not acceptable for action
+                TooOldExpirationDate:
+                  description: The expiration date of the last issued cert is too old.
+                  value:
+                    error: RI400-1600
+                    message: expiration date too old to become extended
+                TooEarlyExpirationDate:
+                  description: The expiration date of the last issued cert is too early.
+                  value:
+                    error: RI400-1700
+                    message: expiration date too young to become extended
```

Comment:
"too young" => "too recent"

for dates, a better opposite of "old" is "recent"

Reply:
@MikeMcC399 sorry, my last changes weren't included. I hope, it's more clear now.

Reply:
@MikeMcC399 I do agree. The message included is though already deployed on our QA stage. Should I fix it as a hotfix? Or should we leave it like it is for now and then fix it in the next release?

Reply:
@lalottacontinua 
It's not critical. "young" is still understandable, even if it is not common usage in this context.

Reply:
Okay, thanks. I will take it into account for our next release.